Fix #16, update description in plugin.xml

undeadcat opened the pull request.

undeadcat commented:
@daedlock, Sorry, I didn't mean to add this new revision to this review. I didn't realise it would be added automatically.
If you know how, you can remove it, I'll merge these two commits and then create a separate PR for the new commit. Just to keep logically separate things separate.

re: publishing a new .zip. 
I'm afraid of getting confused which version is which if we continue publishing .zip files here and on the plugin repository.
If someone's very anxious about getting the latest fix, they can always build from sources, right?

If you know how to propose a system that will keep us from getting confused, please, share your thoughts=)

daedlock commented:
@undeadcat  

> If you know how, you can remove it, I'll merge these two commits and then create a separate PR for the new commit. Just to keep logically separate things separate.


On your fork, you can do the following
1. To add the last commit to another branch for creating another PR
```
git checkout <anotherbranch>
git cherry-pick origin master
```

2. Then, to remove that commit from this PR
```
git checkout origin master
git reset --hard HEAD~ && git push origin master --force
```

___
> I'm afraid of getting confused which version is which if we continue publishing .zip files here and on the plugin repository.

How about creating a version tag for each releasable PR? So, after this PR is merged, we can tag the merge commit with `v1.0.1` for example. Users would then be able to view all versions on https://github.com/styled-components/webstorm-styled-components/releases

undeadcat commented:
@daedlock, Thanks!
I thought there was a less of a direct correlation between commits and github code reviews. That's why I asked. Fixed, added .zip and bumped version

> How about creating a version tag for each releasable PR? 
Sure.
